lib/rules/one-var.js

```diff
@@ -14,16 +15,39 @@
 
 module.exports = function(context) {
 
-    var MODE = context.options[0] || "always";
-    var options = {};
+    var MODE = context.options[0];
 
-    // simple options configuration with just a string or no option
-    if (typeof context.options[0] === "string" || context.options[0] == null) {
-        options.var = MODE;
-        options.let = MODE;
-        options.const = MODE;
-    } else {
-        options = context.options[0];
+    // default options
+    var options = {
+        var: { uninitialized: "always", initialized: "always"},
+        let: { uninitialized: "always", initialized: "always"},
```

Comment:
Probably a good idea to put "always" and "never" into some constants rather than using the same string everywhere.

```
var MODE_ALWAYS = "always",
    MODE_NEVER = "never";
```